apps/system/js/rocketbar.js

```diff
@@ -567,6 +568,14 @@
       if (Service.currentApp.isPrivateBrowser()) {
         this.setInput('');
       }
+
+      // Only record the search if we have internet connectivity
+      if(navigator.onLine) {
```

Comment:
it may not matter, but should we move the online logic into AppUsageMetrics? is there a usecase in the future where we might want to know if a search occurred from the rocketbar, but the phone wasn't online?